docs/docs/quick_start/advanced_features/disk_expansion.md

```diff
@@ -4,3 +4,89 @@ sidebar_label: "Disk Expansion"
 ---
 
 # Disk Expansion
+
+A storage system is usually expected to expand its capacity by adding a new disk into a storage node. In HwameiStor, it can be done with the following steps.
+
+## Steps
+
+### 1. Prepare a new storage disk
+
+Select a storage node from the HwameiStor system, and add a new disk into it.
+
+For example, the storage node and new disk information are as follows:
+
+- name: k8s-worker-4
+- devPath: /dev/sdc
+- diskType: SSD
+
+After the new disk is added into the storage node `k8s-worker-4`, you can check the disk status as below:
+
+```console
+# 1. Check if the new disk is added into the node successfully
+$ ssh root@k8s-worker-4
+$ lsblk | grep sdc
+sdc        8:32     0     20G  1 disk
+
+
+# 2. Check if the LocalDisk CR already exists for the new disk and the status is "Unclaimed"
+$ kubectl get localdisk | grep k8s-worker-4 | grep sdc
+k8s-worker-4-sdc   k8s-worker-4       Unclaimed 
+```
+
+### 2. Add the new disk into the node's storage pool
+
+The new disk should be added into the existing SSD storage pool of the node. If the storage pool doesn't exists, it will be constructed automatically and the new disk should be added into it.
```

Comment:
```suggestion
The new disk should be added into the existing SSD storage pool of the node. If the storage pool doesn't exist, it will be constructed automatically and the new disk should be added into it.
```

Reply:
good catch:)